beetsplug/convert.py

```diff
@@ -399,9 +399,8 @@ def copy_album_art(self, album, dest_dir, path_formats, pretend=False):
                                util.displayable_path(album.artpath),
                                util.displayable_path(dest))
             else:
-                self._log.info(u'Copying cover art to {0}',
-                               util.displayable_path(album.artpath),
-                               util.displayable_path(dest))
+                self._log.info(u'Copying cover art from {0}',
+                               util.displayable_path(album.artpath))
```

Comment:
Length. In my case it's printing the full path, which is quite long, so two of those would wrap the long line and not be very nice in the console. But as you wish, we can also only log to destination !

Reply:
I see! But the other messages here include both paths—for example, see the “Resizing” log message a few lines above. It would be nice if all of these were consistent (either way).